spec/factories/correspondence_types.rb

```diff
@@ -41,8 +40,6 @@
     abbreviation { 'OFFENDER_SAR' }
     escalation_time_limit { 3 }
     internal_time_limit { 10 }
-    external_time_limit { 30 }
-    deadline_calculator_class { 'CalendarDays' }
     report_category_name { 'Offender SAR report' }
 
```

Comment:
Just wondering why we don't have the `deadline_calculator_class { 'CalendarMonth' }` here like the SAR one above?